.mdl_style.rb

```diff
@@ -0,0 +1,8 @@
+# Enable all rules
+all
+
+# We have own check to validate the line length
+exclude_rule 'MD013'
+
+# It is OK to have a plain URL in README
```

Comment:
`We do not use some parsers, our md files are for Github mostly, and it works fine with bare URLs`

Reply:
done